app/main/routes.py

```diff
@@ -359,20 +422,39 @@ def search():
         or request.args.get("query", "").lower()
     )
     page = int(request.args.get("page", 1))
+
     filters = {"query": query}
     sorting_orders = build_sorting_orders(request.args)
 
+    breadcrumb_values = {
+        0: {"transferring_body_id": _id},
+        1: {"transferring_body": Body.query.get(_id).Name},
+        2: {"query": ""},
+        3: {"search_terms": ""},
+    }
+
     if query:
+        str_items = ""
+        query_items = query.strip().split(",")
+        # exclude any empty items
+        query_items = list(filter(None, query_items))
+
+        if len(query_items) > 0:
```

Comment:
worth extracting into its own function but happy for it to be done in followup PR alongside the othe r changes